src/middleware/middleware.ts

```diff
@@ -40,14 +40,14 @@ export function middleware(
     });
   }
 
-  const eventName = request.headers["x-github-event"] as WebhookEvent["name"];
-  const signature = request.headers["x-hub-signature"];
-  const id = request.headers["x-github-delivery"];
+  const eventName = request.headers["x-github-event"] as string;
+  const signature = request.headers["x-hub-signature"] as string;
+  const id = request.headers["x-github-delivery"] as string;
```

Comment:
headers are always strings for us, but Typescript things they can also be `string[]` which makes no sense to me